k9mail/src/main/java/com/fsck/k9/mailstore/LocalStore.java

```diff
@@ -594,9 +596,9 @@ public boolean isCopyCapable() {
      * call the MessageRetrievalListener for each one
      */
     List<LocalMessage> getMessages(
-        final MessageRetrievalListener<LocalMessage> listener,
-        final LocalFolder folder,
-        final String queryString, final String[] placeHolders
+            final MessageRetrievalListener<LocalMessage> listener,
+            final LocalFolder folder,
+            final String queryString, final String[] placeHolders
     ) throws MessagingException {
         final List<LocalMessage> messages = new ArrayList<>();
         final int j = database.execute(false, new DbCallback<Integer>() {
```

Comment:
@Valodim @cketti Any idea why we first query 10 rows and then the remaining ones in this method ?